packages/docregistry/src/context.ts

```diff
@@ -837,7 +837,7 @@ namespace Private {
   export function getSavePath(path: string): Promise<string | undefined> {
     let saveBtn = Dialog.okButton({ label: 'SAVE' });
     return showDialog({
-      title: 'Save File As..',
+      title: 'Save File As...',
```

Comment:
We intend to use the unicode ellipses in these cases, I guess we missed a spot here.